packages/DatePicker/src/DatePicker.styles.js

```diff
@@ -2,6 +2,12 @@ import { css } from "styled-components";
 
 import tokens from "@paprika/tokens";
 
+const opacityNoneStyles = css`
+  opacity: 0;
```

Comment:
Needs to be opacity rather than `visibility` in order for the click to propagate correct and the input getting focus